sailor/assetcentral/group.py

```diff
@@ -0,0 +1,219 @@
+"""
+Group module can be used to retrieve Groups information from AssetCentral.
+
+Classes are provided for individual Groups as well as groups of Groups (GroupSet).
+"""
+
+from functools import cached_property
+import warnings
+
+from .utils import AssetcentralEntity, ResultSet, _ac_application_url, _fetch_data, _apply_filters_post_request, \
+    _add_properties
```

Comment:
for new code we should use the recommended way of doing multi-line imports using parentheses (tuple). I also started using this when I had to change one of these import lines in my former PRs.
https://www.python.org/dev/peps/pep-0328/#rationale-for-parentheses